keycloak/keycloak-admin-ui
# User fed mappers layout and partial logic

## Motivation
https://issues.redhat.com/browse/APPDUX-541
https://issues.redhat.com/browse/APPDUX-707

## Brief Description
Adds mapper layout for new user fed mappers tab and forms for the 11 different mapper types, and fetches values for pre-existing mappers.

## Verification Steps
1. Create an LDAP provider if one doesn't already exist.
2. In the old admin console, add a mapper of each type for the LDAP provider.
3. In the new console, verify that the mappers are listed in the new Mappers tab of the LDAP provider.
4. Click each mapper and verify that the appropriate mapper information appears in its details, when comparing to the old console.

Note: This is part 1 of the user fed mapper implementation, was originally just for the layout, but i've added logic for fetching all the values for pre-existing mappers. Creating new, editing, saving, deleting, etc and other CRUD logic, as well as cypress tests, are all coming in part 2 scheduled for next sprint.

## Checklist:
